soot-infoflow/src/soot/jimple/infoflow/typing/TypeUtils.java

```diff
@@ -176,6 +180,7 @@ public boolean hasCompatibleTypesForCall(AccessPath apBase, SootClass dest) {
 		// Cannot invoke a method on a primitive type
 		if (apBase.getBaseType() instanceof PrimType)
 			return false;
+		FastHierarchy hierarchy = manager.getHierarchy();
```

Comment:
Wofür wird diese Hierarchy benutzt?

Reply:
Habe sie rausgenommen; wollte erst den Check in der Methode machen.